pkg/util/pass-through/pass_through.go

```diff
@@ -4,10 +4,16 @@ type passRequest string
 
 const (
 	versionRequest passRequest = "/version::get"
+	healthRequest  passRequest = "/healthz::get"
```

Comment:
Just a comment, we could leave the comment like `// deprecated: TODO @fujitatomoya remove this once it is gone`.

https://kubernetes.io/docs/reference/using-api/health-checks/

The healthz endpoint is deprecated (since Kubernetes v1.16), so we need to align with k8s once it is removed.